pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -329,26 +337,39 @@ func (o *MustGatherOptions) Run() error {
 		_, err := imagereference.Parse(image)
 		if err != nil {
 			o.log("unable to parse image reference %s: %v", image, err)
+			// ensure the errors bubble up to BackupGathering method for display
+			errs = []error{err}
```

Comment:
might be better to have `"unable to parse image reference %s: %v"` extra info in the error as well instead of just `error: invalid reference format`

Reply:
Done